src/pages/AllProducts.jsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
+import Navbar from '../Components/Navbar'
```

Comment:
Take pull from `develop` branch and use lowercase `components`

Reply:
@saurabhbakolia I have done changes can you please check it out